### DISCUSSION TITLE
zsh: command not found: npm

### DISCUSSION THREAD

**Discussion by Dchristie45** [ohmyzsh/ohmyzsh#9790] [Help] [answered]
Hi, 

I installed the node.js from nodejs.org on my Mac Terminal.  Please see the below messages. 

This package has installed:
-   Node.js v14.16.0 to /usr/local/bin/node
-   npm v6.14.11 to /usr/local/bin/npm

Make sure that /usr/local/bin is is your $PATH

I then went in the terminal to check if node.js was installed by typing the below code: 

davidd.christie@Davids-MBP ~ % node --version
v14.16.0

I then tried to initialize NPM by typing :

davidd.christie@Davids-MBP ~ % npm init

Then, I  got the below message:

zsh: command not found: npm

I am not exactly sure what to do from here.  I have been researching and I keep getting bash information.  I am fairly new to coding and trying to make sense of it all.  I am abit confused.  I don't even know what is in my PATH and what is not.  Would some please help me?  I would greatly appreciate your help.

**Comment by mcornella** [comment, marked as answer]
See https://github.com/ohmyzsh/ohmyzsh/discussions/9736 for pointers to how to solve this.
